mdata/cache/cache_chunk.go

```diff
@@ -0,0 +1,12 @@
+package cache
+
+import (
+	"github.com/raintank/metrictank/mdata/chunk"
+)
+
+type CacheChunk struct {
+	Ts    uint32
+	Next  uint32
+	Prev  uint32
```

Comment:
wouldn't it make more sense to put prev before next? (sorry i just like things in order :) )